presenter/droplet.go

```diff
@@ -0,0 +1,88 @@
+package presenter
+
+import (
+	"fmt"
+
+	"code.cloudfoundry.org/cf-k8s-api/repositories"
+)
+
+type DropletResponse struct {
+	GUID              string            `json:"guid"`
+	CreatedAt         string            `json:"created_at"`
+	UpdatedAt         string            `json:"updated_at"`
+	State             string            `json:"state"`
+	Error             *string           `json:"error"`
+	Lifecycle         Lifecycle         `json:"lifecycle"`
+	ExecutionMetadata string            `json:"execution_metadata"`
+	Checksum          *ChecksumData     `json:"checksum"`
+	Buildpacks        []BuildpackData   `json:"buildpacks"`
+	ProcessTypes      map[string]string `json:"process_types"`
+	Stack             string            `json:"stack"`
+	Image             *string           `json:"image"`
+	Relationships     Relationships     `json:"relationships"`
+	Metadata          Metadata          `json:"metadata"`
+	Links             map[string]*Link  `json:"links"`
+}
+
+type ChecksumData struct {
+	Type  string `json:"type"`
+	Value string `json:"value"`
+}
+
+type BuildpackData struct {
+	Name          string `json:"name"`
+	DetectOutput  string `json:"detect_output"`
+	BuildpackName string `json:"buildpack_name"`
+	Version       string `json:"version"`
+}
+
+func ForDroplet(dropletRecord repositories.DropletRecord, baseURL string) DropletResponse {
+	toReturn := DropletResponse{
+		GUID:      dropletRecord.GUID,
+		CreatedAt: dropletRecord.CreatedAt,
+		UpdatedAt: dropletRecord.UpdatedAt,
+		State:     dropletRecord.State,
+		Lifecycle: Lifecycle{
+			Type: dropletRecord.Lifecycle.Type,
+			Data: LifecycleData{
+				Buildpacks: dropletRecord.Lifecycle.Data.Buildpacks,
+				Stack:      dropletRecord.Lifecycle.Data.Stack,
+			},
+		},
+		ExecutionMetadata: "",
+		Buildpacks:        make([]BuildpackData, 0),
```

Comment:
FYI, this could also be `[]BuildpackData{}`, which I think reads a little better.

Reply:
Addressed on commit [8a5dfced](https://github.com/cloudfoundry/cf-k8s-api/pull/131/commits/8a5dfced2f8a16a0d25310f384e2cb265d2a4a7d)